cmd/shellwrapper.go

```diff
@@ -67,6 +67,26 @@ konf_cleanup() {
   konf-go cleanup
 }
 
+trap konf_cleanup EXIT
```

Comment:
Unfortunately Github does not let me comment on unchanged files, so I am going to do it in here: Could you update the Readme section "### 2. Install the konf shellwrapper" to include instructions how to set it up for fish?

I am super new to fish, but from my current understanding, fish has a unique syntax. So I would propose to update the structure to something like this:

```markdown
### 2. Install the konf shellwrapper

Depending on whether you are using zsh/bash or fish, please use the following:

#### A) zsh/bash

-- current content here

#### B) fish

...
```

Reply:
Done, I also made the same for `### Customizations to Have a Good Time` part